docs/docs/introducing-graphiql.md

```diff
@@ -30,6 +30,16 @@ Make sure to check out the GraphiQL docs in the upper right-hand corner of the I
 
 ![A diagram pointing out where to find the GraphiQl docs](images/graphiql-docs.png)
 
+## Using the GraphiQL Explorer
+
+The GraphiQL Explorer enables you to interactively construct full queries by clicking through available fields and inputs without the repetitive process of typing these queries out by hand.
+
+<iframe class="egghead-video" width={600} height={348} src="https://egghead.io/lessons/gatsby-build-a-graphql-query-using-gatsby-s-graphiql-explorer/embed" />
```

Comment:
```suggestion
<iframe class="egghead-video" width={600} height={348} src="https://egghead.io/lessons/gatsby-build-a-graphql-query-using-gatsby-s-graphiql-explorer/embed" title="Video: Build a GraphQL Query using Gatsby’s GraphiQL Explorer" />
```

Reply:
Sounds good! Btw, this is how the other Egghead embeds are on the site rn. I plan to pull this out into an MDX component in the future!